examples/Getting Started with Boutiques.ipynb

```diff
@@ -0,0 +1,351 @@
+      "  -e          generates brain surface as mesh in .vtk format\n",
+      "\n",
+      "Variations on default bet2 functionality (mutually exclusive options):\n",
+      "  (default)   just run bet2\n",
+      "  -R          robust brain centre estimation (iterates BET several times)\n",
+      "  -S          eye & optic nerve cleanup (can be useful in SIENA)\n",
+      "  -B          bias field & neck cleanup (can be useful in SIENA)\n",
+      "  -Z          improve BET if FOV is very small in Z (by temporarily padding end slices)\n",
+      "  -F          apply to 4D FMRI data (uses -f 0.3 and dilates brain mask slightly)\n",
+      "  -A          run bet2 and then betsurf to get additional skull and scalp surfaces (includes registrations)\n",
+      "  -A2 <T2>    as with -A, when also feeding in non-brain-extracted T2 (includes registrations)\n",
+      "\n",
+      "Miscellaneous options:\n",
+      "  -v          verbose (switch on diagnostic messages)\n",
+      "  -h          display this help, then exits\n",
+      "  -d          debug (don't delete temporary intermediate images)\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%bash\n",
+    "bet -h"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Looking at all of these flags, we see a list of options which can be summarized by:\n",
+    "```\n",
+    "bet [INPUT_FILE] [MASK] [FRACTIONAL_INTENSITY] [VERTICAL_GRADIENT] [CENTER_OF_GRAVITY] [OVERLAY_FLAG] [BINARY_MASK_FLAG] [APPROX_SKULL_FLAG] [NO_SEG_OUTPUT_FLAG] [VTK_VIEW_FLAG] [HEAD_RADIUS] [THRESHOLDING_FLAG] [ROBUST_ITERS_FLAG] [RES_OPTIC_CLEANUP_FLAG] [REDUCE_BIAS_FLAG] [SLICE_PADDING_FLAG] [MASK_WHOLE_SET_FLAG] [ADD_SURFACES_FLAG] [ADD_SURFACES_T2] [VERBOSE_FLAG] [DEBUG_FLAG]\n",
+    "```\n",
+    "\n",
+    "Now that we have summarized all command line options for our tool - some of which describe inputs and others, outputs - we can begin to craft our JSON Boutiques identifier."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 2: Understanding Boutiques + JSON\n",
+    "\n",
+    "For those unfamiliar with JSON, we recommend following this [3 minute JSON tutorial](http://www.secretgeek.net/json_3mins) to get you up to speed. In short, a JSON file is a dictionary object which contains *keys* and associated *values*. A *key* informs us what is being described, and a *value* is the description (which, importantly, can be arbitrarily typed). The Boutiques identifier is a JSON file which requires the following keys, or, properties:\n",
+    "- `name`\n",
+    "- `description`\n",
+    "- `schema-version`\n",
+    "- `command-line`\n",
+    "- `inputs`\n",
+    "- `output-files`\n",
+    "\n",
+    "Some additional, optional, properties that a Boutiques fill will recognize are:\n",
+    "- `groups`\n",
+    "- `tool-version`\n",
+    "- `docker-image`\n",
+    "- `docker-index`\n",
```

Comment:
Should be adjusted to the new 0.4 schema:
"container-image": {
  "type": "docker",
  "image": "someimage",
  "index": "someindex"
}